blueprint/requirements.txt

```diff
@@ -0,0 +1,4 @@
+invoke==1.7.1
+plasTeX @ git+https://github.com/plastex/plastex.git
+leanblueprint @ git+https://github.com/utensil/leanblueprint.git@lean4-only-dev
```

Comment:
This is currently depending on my `dev` branch, will change it to `lean4-only` before merge.